src/Tribe/Main.php

```diff
@@ -2988,15 +2988,18 @@ public static function activate() {
 				return;
 			}
 
-			self::flushRewriteRules();
-
 			if ( ! class_exists( 'Tribe__Events__Editor__Compatibility' ) ) {
 				require_once $plugin_path . '/src/Tribe/Editor/Compatibility.php';
 			}
 
 			$editor_compatibility = new Tribe__Events__Editor__Compatibility();
 			$editor_compatibility->deactivate_gutenberg_extension_plugin();
 
+			if ( ! is_network_admin()  ) {
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Expected exactly one space before closing parenthesis; "  " found.

<sub>hash: 7f660b046c72941d70ef4cf2b2a6e513</sub>